src/parser/jobs/brd/modules/index.ts

```diff
@@ -15,5 +14,4 @@ export default [
 	OGCDDowntime,
 	RagingStrikes,
 	RefulgentProcs,
```

Comment:
I know you weren't otherwise touching Raging Strikes and Refulgent Procs on this PR, but it would be nice if we could fix up the `export default`s on those too while we're at it